meta-lmp-bsp/conf/machine/uz3eg-iocc.conf

```diff
@@ -12,3 +12,7 @@ HDF_MACHINE ?= "zcu102-zynqmp"
 HDF_BASE = "file://"
 HDF_PATH = "system.dsa"
 HDF_EXT = "dsa"
+
+# bitstream support
+MACHINE_FIRMWARE_append_uz3eg-iocc = " bitstream-extraction fpga-manager-script"
```

Comment:
No need for append_uz3eg-iocc as this is already the board configuration file, just append works.